lib/color_guard.rb

```diff
@@ -29,6 +29,10 @@ def deactivate!(feature_name)
       end
     end
 
+    def all
+      store.all
+    end
```

Comment:
can you rename this `#all_features`? `#all` is nice because its short but its not obviously to me "all what?"